x-pack/plugin/sql/src/main/java/org/elasticsearch/xpack/sql/expression/function/scalar/math/MathFunction.java

```diff
@@ -44,7 +44,7 @@ public String processScript(String template) {
         return super.processScript(format(Locale.ROOT, "{sql}.%s(%s)", mathFunction(), template));
     }
 
-    protected String mathFunction() {
+    private String mathFunction() {
```

Comment:
Let's remove this method all together and simply use the classname inside the template.